src/main/java/seedu/student/commons/core/Messages.java

```diff
@@ -6,14 +6,15 @@
 public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
-    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_NONEXISTENT_MATRIC_NUM = "The matriculation number provided does not exist";
-    public static final String MESSAGE_STUDENTS_ARE_LISTED = "All students are listed";
-    public static final String MESSAGE_NO_STUDENTS_ARE_LISTED = "No students are listed";
-    public static final String MESSAGE_APPOINTMENT_IS_LISTED = " along with an appointment";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format. \n%1$s";
+    public static final String MESSAGE_NONEXISTENT_MATRIC_NUM = "The matriculation number provided does not exist.";
+    public static final String MESSAGE_STUDENTS_ARE_LISTED = "All students are listed.";
+    public static final String MESSAGE_NO_STUDENTS_ARE_LISTED = "No students are listed.";
     public static final String MESSAGE_STUDENTS_AND_APPOINTMENT_FOUND =
-            "Found a student with matriculation number %s \n"
-                    + "Their appointment will also be listed";
-    public static final String MESSAGE_NONEXISTENT_APPOINTMENT = "No appointment is found!";
+            "Found student with matriculation number %s \n"
+                    + "If they have an appointment, their appointment will also be listed.";
```

Comment:
Yeah this line I feel you should change to "No appointment found for this student". So it tells the user that the student exists, but no appointment is found for that student

Reply:
To clarify, the whole MESSAGE_NONEXISTENT_APPOINTMENT line was deleted because it was not used anywhere (greyed out). The "Found student with ... also be listed" string is for MESSAGE_STUDENTS_AND_APPOINTMENT_FOUND.